rts/Lua/LuaSyncedCtrl.cpp

```diff
@@ -3529,12 +3530,34 @@ int LuaSyncedCtrl::SetUnitPosErrorParams(lua_State* L)
 
 	unit->nextPosErrorUpdate = luaL_optint(L, 8, unit->nextPosErrorUpdate);
 
-	if (lua_isnumber(L, 9) && lua_isboolean(L, 10))
-		unit->SetPosErrorBit(std::clamp(lua_tointeger(L, 9), 0, teamHandler.ActiveAllyTeams()), lua_toboolean(L, 10));
```

Comment:
Here

Reply:
Right XD, anyways I don't see the point of creating new methods just for setting smth that makes thing work badly tbh.

Reply:
The functionality solves my use case and I am happy with it as a game dev.

Reply:
But not the nanoturrets problem, what use case is that?

Reply:
It is the nanoturrets problem. I don't know what issues you see with it. https://github.com/ZeroK-RTS/Zero-K/blob/master/LuaRules/Gadgets/unit_radar_wobble.lua#L32-L45

Reply:
What can I say, to me looks very bad when the engine could do it right on its own.

Reply:
It could not do it right on its own because it is up to a game dev to decide what is right and what is wrong. At most it can provide reasonable defaults. I agree that making nanos not wobble would be a reasonable default though.

Reply:
Exactly, and we can do that, just need to decide how we want to do it, just checking non moving builder or have some customparam or unitdefparam. (i guess checking for non moving builder)

Reply:
Also.. do we have an issue of why nanoturret has to be a unit? Would like to know all the info and maybe look for solutions too.

Reply:
Nanos could be a building if there was a nice way of distinguishing them from a factory.

Reply:
> Nanos could be a building if there was a nice way of distinguishing them from a factory.

Need more information, that doesn't really tell me a lot about what's going on, like... they can be distinguished with an unitdef param or whatever.

Reply:
> Also.. do we have an issue of why nanoturret has to be a unit? Would like to know all the info and maybe look for solutions too.

#1597